app/src/main/java/com/codesoom/assignment/application/TaskService.java

```diff
@@ -24,13 +26,14 @@ public Task getTask(Long id) {
     }
 
     public Task createTask(Task source) {
-        Task task = new Task();
-        task.setId(generateId());
-        task.setTitle(source.getTitle());
+        if(source.getTitle() == null){
+            throw new TaskNotCreateException(source.getId());
```

Comment:
그러고보니 이 예외 클래스는 이름을 이렇게 바꾸는 것도 괜찮겠어요.
```suggestion
            throw new TaskCreationFailException(source.getId());
```